cli/templates/config.jinja2

```diff
@@ -3,3 +3,4 @@ GRPC_ADDRESS = {
     'port': 50051,
 }
 GRPC_LOGGER = 'rpc'
+CHANNEL_OPTIONS = None
```

Comment:
Prev settings name starts with 'GRPC_', maybe `CHANNEL_OPTIONS` should starts with 'GRPC_' too.